terraform/cyhy_nmap_ec2.tf

```diff
@@ -104,7 +104,7 @@ resource "aws_ebs_volume" "nmap_cyhy_runner_data" {
   availability_zone = "${var.aws_region}${var.aws_availability_zone}"
   # availability_zone = "${element(data.aws_availability_zones.all.names, count.index)}"
   type = "gp2"
-  size = "${local.production_workspace ? 128 : 1}"
+  size = "${local.production_workspace ? 2 : 1}"
```

Comment:
Again, is 2GB the value we used before?

Reply:
2GB is what [we used to have for this;](https://github.com/dhs-ncats/cyhy_amis/commit/b0b6b4175d080ced414494fe8691f03405b41207#diff-0ec35a026f754ebbbc04103a8e4c9031) the runner doesn't require that much space if we only have 12 concurrent jobs running on each portscan instance.